karapace/typing.py

```diff
@@ -1,11 +1,10 @@
-from typing import Any, Dict, Union
+from typing import Any, Union
 
 JsonData = Any  # Data that will be encoded to or has been parsed from JSON
 
 Subject = str
 
-# Container type for a subject, with configuration settings and all the schemas
-SubjectData = Dict[str, Any]
-
 Version = Union[int, str]
 ResolvedVersion = int
+
+SchemaId = int
```

Comment:
issue: In general I'm skeptical to this usage of type aliases, as I think it can be deceptive and giving the impression that there is type safety when there isn't. Can we use a `NewType` here instead?

Reply:
Typing should be separate refactoring. Current usage is to highlight places in code where schema id is used.